### PR TITLE
Only log in debug in samples

### PR DESCRIPTION
**Pull request by oldalton**
It's not the best practice to always NSLog identity logs in samples.

### CODE REVIEW

File: Samples/ios/SampleApp/SampleAppiOS/SampleMSALUtil.m [hunk 1]
```diff
@@ -64,7 +64,11 @@ + (void)setup
         {
```
Comment (jasoncoolmax) [inline Samples/ios/SampleApp/SampleAppiOS/SampleMSALUtil.m]: Seems line 57-60 is no longer true? :)

File: Samples/ios/SampleAppiOS-Swift/SampleAppiOS-Swift/SampleMSALAuthentication.swift [hunk 2]
```diff
@@ -48,7 +48,11 @@ class SampleMSALAuthentication {
             // about the account being logged in.
             if let displayableMessage = message {
                 if (!containsPII) {
+#if DEBUG
```
Comment (jasoncoolmax) [inline Samples/ios/SampleAppiOS-Swift/SampleAppiOS-Swift/SampleMSALAuthentication.swift]: Line 45-46 is no longer true? :)

Comment (oldalton) [reply Samples/ios/SampleAppiOS-Swift/SampleAppiOS-Swift/SampleMSALAuthentication.swift]: updated :)

File: Samples/ios/SampleApp/SampleAppiOS/SampleMSALUtil.m (hunk 1 above)
Comment (oldalton) [reply Samples/ios/SampleApp/SampleAppiOS/SampleMSALUtil.m]: updated :)

**Pull request merged**